mkosi/initrd.py

```diff
@@ -21,43 +21,54 @@
 from mkosi.util import resource_path
 
 
-@uncaught_exception_handler()
-def main() -> None:
-    log_setup()
-
-    parser = argparse.ArgumentParser(
-        prog="mkosi-initrd",
-        description="Build initrds or unified kernel images for the current system using mkosi",
-        allow_abbrev=False,
-        usage="mkosi-initrd [options...]",
+def initrd_copy_host_config(staging_dir: str) -> list[str]:
```

Comment:
Let's handle crypttab and kernel/cmdline separately. (Don't necessarily need a separate function for kernel/cmdline.

So I'd rename this to `process_crypttab()` or so